packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -675,6 +675,13 @@ class _FocusState extends State<Focus> {
     Widget child = widget.child;
     if (widget.includeSemantics) {
       child = Semantics(
+        onDidGainAccessibilityFocus: () {
```

Comment:
It will be useful for a future reader of this code to see an explanation of why nothing is done about `onDidLoseAccessibilityFocus`.

Reply:
Done.